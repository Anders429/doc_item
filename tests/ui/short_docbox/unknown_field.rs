use doc_item::short_docbox;

#[short_docbox(invalid="field")]
fn unknown_field() {}

fn main() {}
