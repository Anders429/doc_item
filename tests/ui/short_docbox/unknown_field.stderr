error: Unknown field: `invalid`
 --> $DIR/unknown_field.rs:3:16
  |
3 | #[short_docbox(invalid="field")]
  |                ^^^^^^^^^^^^^^^
